app/Config/Migrations.php

```diff
@@ -44,7 +44,10 @@ class Migrations extends BaseConfig
      * using the CLI command:
      *   > php spark migrate:create
      *
-     * Typical formats:
+     * Note: if you set an unsupported format, migration runner will not find
+     *       your migration files.
+     *
+     * Supported formats:
      * - YmdHis_
      * - Y-m-d-His_
      * - Y_m_d_His_
```

Comment:
When format `Y_m_d_His_` is used, there are problems in displaying the file name in command `php spark migrate:status`.

```console
P:\Bug\project-root>php spark migrate:create
CodeIgniter v4.2.10 Command Line Tool - Server Time: 2022-12-28 22:15:58 UTC-06:00

Migration class name : KenjisTest

File created: APPPATH\Database\Migrations\2022_12_29_041610_KenjisTest.php

P:\Bug\project-root>php spark migrate:create
Migration class name : PooyaTest
File created: APPPATH\Database\Migrations\2022_12_29_041619_PooyaTest.php

P:\Bug\project-root>php spark migrate:status

CodeIgniter v4.2.10 Command Line Tool - Server Time: 2022-12-28 22:16:30 UTC-06:00

+----------------------+-------------------+------------------------+-------+-------------+-------+
| Namespace            | Version           | Filename               | Group | Migrated On | Batch |
+----------------------+-------------------+------------------------+-------+-------------+-------+
| App                  | 2022_12_29_041610 | 0_KenjisTest           | ---   | ---         | ---   |
| App                  | 2022_12_29_041619 | 12_29_041619_PooyaTest | ---   | ---         | ---   |
| Datamweb\ShieldOAuth | 2022-10-20-182737 | ShieldOAuth            | ---   | ---         | ---   |
| CodeIgniter\Shield   | 2020-12-28-223112 | create_auth_tables     | ---   | ---         | ---   |
+----------------------+-------------------+------------------------+-------+-------------+-------+
```
why `0_KenjisTest ` and `12_29_041619_PooyaTest`?

Reply:
```console
+-----------+-------------------+-------------------------+-------+-------------+-------+
| Namespace | Version           | Filename                | Group | Migrated On | Batch |
+-----------+-------------------+-------------------------+-------+-------------+-------+
| App       | 2022_12_29_050744 | 12_29_050744_KenjisTest | ---   | ---         | ---   |
| App       | 2022_12_29_050800 | 12_29_050800_PooyaTest  | ---   | ---         | ---   |
+-----------+-------------------+-------------------------+-------+-------------+-------+
```

Reply:
![bug-cli](https://user-images.githubusercontent.com/9530214/209906626-20c244b6-5515-4f79-b3fa-b37f4ed01337.jpg)
@kenjis question is, `12_29_050744_KenjisTest` or `KenjisTest`, which one is correct?

Reply:
`KenjisTest` is correct. Because the migration filename is made of *number* `_` *classname*.

This is a bug. I will fix it.
But I don't know why you got `0_KenjisTest `.

Reply:
@datamweb I sent PR: #7038
Try it when you have time.

Reply:
> But I don't know why you got `0_KenjisTest `.

As it is clear, the output was in Figure `0_KenjisTest`. In this regard, I am sure that the received output was `0_KenjisTest`.
At the moment I can not reproduce.